docs/actions-pipelines.md

```diff
@@ -72,6 +72,14 @@ In general, actions are composed as follows:
 * Each action must include an `outputs` key with at least one output, classified as either `highly_sensitive` or `moderately_sensitive`
     * `highly_sensitive` outputs are considered potentially highly-disclosive, and are never intended for publishing outside the secure environment
     * `moderately_sensitive` outputs are automatically copied to the secure review area for redaction (otherwise known as [Level 4](security-levels.md)) and potentially for publication back to GitHub.
+    * Outputs should be separated onto different lines, each with a unique 'key', but related outputs can be combined using a wildcard (`*`). E.g.:
+        ```yaml
+           outputs:
+              moderately_sensitive:
+                table: output/summary_results.txt
+	            survival_figure: output/figures/survival-plot.png
+	            time_series_figures: output/figures/time_series_*.png
```

Comment:
The last two lines of your example are indented with tabs. Please could you change these to multiple spaces? That way we can be sure that it all lines up whatever the tab indentation is set to in people's editors. (It doesn't affect how this is ultimately rendered, so this is just a nicety for other people editing the docs.)

Reply:
Aha I didn't realise there were tabs there, I wondered why it was lining up in strange ways